doc/manual/src/release-notes/rl-next.md

```diff
@@ -1,2 +1,6 @@
 # Release X.Y (202?-??-??)
 
+- Speed-up of downloads from binary caches.
+  The number of parallel downloads (also known as substitutions) has been separated from the [`--max-jobs` setting](../command-ref/conf-file.md#conf-max-jobs).
+  The new setting is called [`max-substitution-jobs`](../command-ref/conf-file.md#conf-max-substitution-jobs).
+  The number of parallel downloads is now set to 16 by default (before this change the default was set to 1).
```

Comment:
```suggestion
  The number of parallel downloads is now set to 16 by default (previously, the default was 1 due to the coupling to build jobs).
```

This should make it slightly clearer, I had to parse it multiple times before that.